install/kubernetes/cilium/templates/clustermesh-apiserver/clusterrole.yaml

```diff
@@ -34,17 +44,23 @@ rules:
   - customresourcedefinitions
   verbs:
   - list
+  - watch
 - apiGroups:
-  - cilium.io
+  - ""
   resources:
-  - ciliumnodes
-  - ciliumnodes/status
-  - ciliumexternalworkloads
-  - ciliumexternalworkloads/status
-  - ciliumidentities
-  - ciliumidentities/status
-  - ciliumendpoints
-  - ciliumendpoints/status
+  - endpoints
+  - namespaces
+  - services
   verbs:
-  - '*'
+  - get
+  - list
+  - watch
+- apiGroups:
+  - discovery.k8s.io
+  resources:
+  - endpointslices
```

Comment:
Should this also contain `endpoints` or are we assuming that `endpointslices` support is enabled everywhere by this point? (possibly more relevant to backports)

Reply:
endpoints is on line 51